script.service.janitor/utils.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+import re
+import time
+from ctypes import *
+
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcvfs
+from settings import *
+
+# Addon info
+ADDON_ID = "script.service.janitor"
+ADDON = xbmcaddon.Addon(ADDON_ID)
```

Comment:
please use only `xbmcaddon.Addon()`. The id is only needed if you're trying to access another plugin